.github/workflows/workflow-executor.yaml

```diff
@@ -48,6 +48,22 @@ on:
         description: "Version to manually set for SDK generation"
         required: false
         type: string
+      env_vars:
```

Comment:
is `environment` not allowed. Not sure if these contraction border on colloquial.

Reply:
Should be. Happy to go with that.